src/pwa-app/src/Layouts/BLayout.jsx

```diff
@@ -37,6 +39,9 @@ const useStyles = makeStyles((theme) => ({
         top: 'auto',
         paddingTop: "2em"
     },
+    main: {
+        paddingTop: "10em",
```

Comment:
it will look like this if i delete them: 
![image](https://user-images.githubusercontent.com/70498757/109125246-66403280-777e-11eb-9f62-0a9c90e6e935.png)

Reply:
ok please handle for me @ducquang2